__tests__/model/model.mutation.test.js

```diff
@@ -28,254 +34,264 @@ describe('Product queries', () => {
+              }
+              category {
                 name {
-                    value
-                    lang
-                }
-                description {
-                    value
-                    lang
-                }
-                images {
-                    large 
-                    medium
-                    small
-                    thumbnail
-                }
-                category {
-                    name { 
-                        value
-                    }
+                  value
                 }
               }
-              ...on Error {
-                  statusCode
-                  message
-              }
+            }
+            ... on Error {
+              statusCode
+              message
             }
           }
-        `,
-        variables: { model: { ...newModelMutation, category: categoryId } },
-      });
+        }
+      `,
+      variables: { model: { ...newModelMutation, category: categoryId } },
+    });
 
-      const error = res.data.addModel;
-      expect(error).toBeDefined();
-      expect(error).toHaveProperty('statusCode', 400);
-      expect(error).toHaveProperty('message', 'MODEL_ALREADY_EXIST');
+    const error = res.data.addModel;
+    expect(error).toBeDefined();
+    expect(error).toHaveProperty('statusCode', 400);
+    expect(error).toHaveProperty('message', 'MODEL_ALREADY_EXIST');
   });
   test('Should throw error MODEL_NOT_FOUND', async () => {
     const res = await client.mutate({
-        mutation: gql`
-          mutation($id: ID!) {
-            deleteModel(id: $id) {
-              ... on Model {
+      mutation: gql`
+        mutation($id: ID!) {
+          deleteModel(id: $id) {
+            ... on Model {
+              name {
+                value
+                lang
+              }
+              description {
+                value
+                lang
+              }
+              images {
+                large
+                medium
+                small
+                thumbnail
+              }
+              category {
                 name {
-                    value
-                    lang
-                }
-                description {
-                    value
-                    lang
-                }
-                images {
-                    large
-                    medium
-                    small
-                    thumbnail
+                  value
                 }
-                category {
-                    name { 
-                        value
-                    }
-                }
-              }
-              ... on Error {
-                statusCode
-                message
               }
             }
+            ... on Error {
+              statusCode
+              message
+            }
           }
-        `,
-        variables: { id: "56ade69dd46eafc5968e5390" },
-      });
+        }
+      `,
+      variables: { id: '56ade69dd46eafc5968e5390' },
```

Comment:
Please move this id to const wrongId = '56ade69dd46eafc5968e5390' and use it here

Reply:
fixed